### PR TITLE
Additional os_firewall role refactoring

### PR DESCRIPTION
**Pull request by mtnbikenc**
* Remove openshift_facts dependency
* Move firewall initialization from std_include.yml to openshift_cluster/config.yml

Installing firewall packages is only necessary during OpenShift installation.

### CODE REVIEW

**Comment by mtnbikenc** [comment]
aos-ci-test

File: roles/os_firewall/tasks/iptables.yml [hunk 1]
```diff
@@ -15,11 +15,13 @@
   when: task_result | changed
 
 - name: Install iptables packages
-  package: name={{ item }} state=present
+  package:
+    name: item
```
Comment (sdodson) [inline roles/os_firewall/tasks/iptables.yml]: "{{ item }}"

**Comment by mtnbikenc** [comment]
aos-ci-test

**Comment by mtnbikenc** [comment]
[merge]

**Comment by mtnbikenc** [comment]
Rebased with recent additions to master.

**Comment by mtnbikenc** [comment]
aos-ci-test

**Comment by sdodson** [comment]
FWIW, the bots should always checkout master and then merge the PR on top when running their tests so unless there's a conflict we shouldn't have to worry about rebasing a PR.

**Comment by mtnbikenc** [comment]
@sdodson for this particular one it was flagged by github as having merge conflicts.  That's the only reason I rebased.  I was just quicker than the bot getting the `needs-rebase` label on the PR.

**Pull request merged**